.github/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -e
+
+if [[ "${OS_NAME}" == "macos-latest" ]]; then
+    brew install \
+        checkbashisms \
+        qpdf
+elif [[ "${OS_NAME}" == "ubuntu-latest" ]]; then
+    sudo apt-get update
+    sudo apt-get install \
+        --no-install-recommends \
+        -y \
+        --allow-downgrades \
+            libcurl4-openssl-dev \
+            libfribidi-dev \
+            libharfbuzz-dev \
+            curl \
+            devscripts \
+            qpdf \
+        || exit -1
+fi
```

Comment:
Did you check if we need these dependencies? I would've expected that between `r-lib/actions/setup-r`, `r-lib/actions/setup-pandoc`, and `r-lib/actions/setup-tinytex` we'd be covered. If it's the case that we don't need this bash script, you could move the `remotes` stuff into `ci.yml` under run.

Reply:
I looked around tonight and found that `r-lib/actions/setup-r` now  seems to install  `qpdf`  and  `checkbashisms` for you (it used  to not), so I'll try removing those.

But some of these are needed by other packages in `{pkgnet}`'s dependencies. Only necessary on Linux because CRAN doesn't prepare precompiled binaries on Linux, so all those things need to be built from source.

```text
libcurl4-openssl-dev \
libfribidi-dev \
libharfbuzz-dev \
curl \
```